### ISSUE TITLE
Query exception when the `users` table has no `name` column

### ISSUE THREAD

**Issue by tkuijer** [laravel/pulse#194]
### Pulse Version

v1.0.0-beta5

### Laravel Version

10.35.0

### PHP Version

8.3.0

### Livewire Version

v3.2.6

### Database Driver & Version

MySQL 8.0.27 MacOS Intel

### Description

In our application the actual name of the user has been moved away from the User model, this results in a query exception since Pulse is now unable to find the `name` column.

### Steps To Reproduce

1. Have a users table without 'name' column.

https://github.com/tkuijer/pulse-bug-report

**Comment by driesvints** [comment]
You always need a name. You can create an accessor on your model for this and pass in the value that you see fit.

**Comment by tkuijer** [comment]
You are correct, this appears to have been fixed in https://github.com/laravel/pulse/commit/16ad1312e0c579e083bde29732c03f006e090254 which changes the way the user data is retrieved.

**Comment by timacdonald** [comment]
You may also configure how users are resolved if you have special needs.

https://laravel.com/docs/10.x/pulse#application-usage-card

<img width="727" alt="Screenshot 2023-12-12 at 9 32 50 am" src="https://github.com/laravel/pulse/assets/24803032/b476d365-7953-48d5-af32-a69060988c16">
